src/main/java/com/wdtinc/mapbox_vector_tile/adapt/jts/JtsAdapter.java

```diff
@@ -512,14 +499,9 @@ public static TileGeomResult createTileGeom(List<Geometry> g,
         final int repeatEndCoordCount = countCoordRepeatReverse(geomCoords);
         final int minExpGeomCoords = geomCoords.length - repeatEndCoordCount;
 
-        // Guard/Optimization: Not enough geometry coordinates for a line
-        if(minExpGeomCoords < 2) {
-            Collections.emptyList();
-        }
-
-
         /** Tile commands and parameters */
-        final List<Integer> geomCmds = new ArrayList<>(geomCmdBuffLenLines(minExpGeomCoords, closeEnabled));
```

Comment:
Can we discard this change? Is it just formatting?

Reply:
happy to do this - the Google checks have a 100 character line limit.